website/docusaurus.config.js

```diff
@@ -51,6 +51,7 @@ module.exports = {
             type: 'all',
             copyright: `Copyright © ${new Date().getFullYear()} Facebook, Inc.`,
           },
+          blogSiteDescription: 'Test description....',
```

Comment:
remove to be able to merge

Reply:
Should I leave blogDescription inside blog config, or should I move it to the root?

Reply:
No it's in the blog options, because a Docusaurus might support hosting multiple blogs in the future, each one having its own description